evennia/contrib/events/tests.py

```diff
@@ -58,7 +58,6 @@ def test_start(self):
         self.assertEqual(self.handler.db.to_valid, [])
         self.assertEqual(self.handler.db.locked, [])
         self.assertEqual(self.handler.db.tasks, {})
-        self.assertEqual(self.handler.db.task_id, 0)
```

Comment:
Why remove this unittest? Should not the sequence of task IDs be fully deterministic?